documentation/api/query/v4/upgrading-from-v3.markdown

```diff
@@ -0,0 +1,128 @@
+---
+title: "PuppetDB 3.0 » API » v4 » Upgrading to PuppetDB 3.0"
+layout: default
+canonical: "/puppetdb/latest/api/query/v4/upgrading-from-v3.html"
+---
+
+This document describes changes that users will need to be aware of
+to make their code compliant with the breaking changes in PuppetDB 3.0. Most of these
```

Comment:
Lets not use the term breaking :-). 'This document describes changes that users will need to be aware of to make their code compliant with version of the query API.'